From 6f9519b001a608af20787e0cfaf4a3a0d2dfc1fb Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Wed, 24 Sep 2014 23:46:00 +0200
Subject: [PATCH] dialog: set again the dialog ids in context for BYE
 processing

- they can be reset by event route execution

(cherry picked from commit 5b733fa361afced0418947c965b256dbe7ae3a68)
---
 modules/dialog/dlg_handlers.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/dialog/dlg_handlers.c b/modules/dialog/dlg_handlers.c
index 45292b3..9ccdfe8 100644
--- a/modules/dialog/dlg_handlers.c
+++ b/modules/dialog/dlg_handlers.c
@@ -1280,6 +1280,7 @@ void dlg_onroute(struct sip_msg* req, str *route_params, void *param)
 
 		_dlg_ctx.cpid = my_pid();
 		_dlg_ctx.expect_t = 1;
+		dlg_set_ctx_iuid(dlg);
 
 		if_update_stat( dlg_enable_stats, active_dlgs, -1);
 		goto done;
-- 
2.1.1

